tests/roxctl/bats-tests/local/central-generate-psps.bats

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bats
```

Comment:
[Not about this file]
When going through code I noticed that Helm tests are missing to complete coverage for PSP cases.
Do you mind checking and adding tests to the following files (should be very straightforward)?

* `pkg/helm/charts/tests/centralservices/testdata/helmtest/central.test.yaml`
* `pkg/helm/charts/tests/centralservices/testdata/helmtest/scanner.test.yaml`